### PR TITLE
gh-2090 Use correct owner key badge icon

### PR DESCRIPTION
**Pull request by elgatovital**
Handles #2090

### CODE REVIEW

File: Multisig/UI/Settings/OwnerKeyManagement/ImportOwnerKey/KeyPickerController/KeyPickerController.swift [hunk 1]
```diff
@@ -79,7 +79,7 @@ class KeyPickerController: UITableViewController {
         vc.trackingEvent = .enterKeyName
         vc.placeholder = "Enter name"
         vc.address = privateKey.address
-        vc.badgeName = KeyType.deviceImported.imageName
+        vc.badgeName = KeyType.deviceGenerated.imageName
```
Comment (MouazAlzahabi) [inline Multisig/UI/Settings/OwnerKeyManagement/ImportOwnerKey/KeyPickerController/KeyPickerController.swift]: I would use .walletConnect type instead

File: Multisig/UI/Settings/OwnerKeyManagement/WalletConnectOwnerKey/OnboardingConnectOwnerKeyViewController.swift [hunk 2]
```diff
@@ -46,7 +46,7 @@ class OnboardingConnectOwnerKeyViewController: AddKeyOnboardingViewController {
             enterNameVC.placeholder = "Enter name"
             enterNameVC.name = connection.remotePeer?.name
             enterNameVC.address = address
-            enterNameVC.badgeName = KeyType.deviceImported.imageName
+            enterNameVC.badgeName = KeyType.walletConnect.imageName
```
Comment (MouazAlzahabi) [inline Multisig/UI/Settings/OwnerKeyManagement/WalletConnectOwnerKey/OnboardingConnectOwnerKeyViewController.swift:49]: Same here

**Review by MouazAlzahabi — CHANGES_REQUESTED** [review]

File: Multisig/UI/Settings/OwnerKeyManagement/ImportOwnerKey/KeyPickerController/KeyPickerController.swift (hunk 1 above)
Comment (elgatovital) [reply Multisig/UI/Settings/OwnerKeyManagement/ImportOwnerKey/KeyPickerController/KeyPickerController.swift]: This screen opens for keys, that were generated on device from a seed phrase. Why use .walletConnect type here?

File: Multisig/UI/Settings/OwnerKeyManagement/WalletConnectOwnerKey/OnboardingConnectOwnerKeyViewController.swift (hunk 2 above)
Comment (elgatovital) [reply Multisig/UI/Settings/OwnerKeyManagement/WalletConnectOwnerKey/OnboardingConnectOwnerKeyViewController.swift:49]: already has .walletConnect type. Am I missing something?

File: Multisig/UI/Settings/OwnerKeyManagement/ImportOwnerKey/KeyPickerController/KeyPickerController.swift (hunk 1 above)
Comment (DmitryBespalov) [reply Multisig/UI/Settings/OwnerKeyManagement/ImportOwnerKey/KeyPickerController/KeyPickerController.swift]: why this change? this should have been deviceImported, this key is imported from a seed phrase or a private key. It's not a wallet connect type.

File: Multisig/UI/Settings/OwnerKeyManagement/WalletConnectOwnerKey/OnboardingConnectOwnerKeyViewController.swift (hunk 2 above)
Comment (DmitryBespalov) [reply Multisig/UI/Settings/OwnerKeyManagement/WalletConnectOwnerKey/OnboardingConnectOwnerKeyViewController.swift:49]: this is correct change.

**Review by DmitryBespalov — APPROVED** [review]

**Review by MouazAlzahabi — APPROVED** [review]

**Pull request merged**